### ISSUE TITLE
Option to save PNG images instead of EXR

### ISSUE THREAD

**Issue by lloydowen8** [lloydowen8/place-heatmap-2022#3]
This was removed just for easy as it had to be rerun on canvas to collected all the data properly. 

Ideally the code should be able to be configured so that it can save the heatmap images as: 

0.png

 ...

 {end_og_canvas}.png

---------------------------

 0-{end_og_canvas+1}.png # old portion of new canvas 

 ... 

 0-{end_files}.png

---------------------------

 1-{end_og_canvas+1}.png # new portion of new canvas
 ... 

1-{end_files}.png

**Comment by memmam** [comment]
Can you clarify what you mean by this? I think I understand what you're getting at, but the way you described the file names is a little confusing. I can look into this later this week, as I'm already looking into how to make the notebook save PNGs instead of EXRs (which seems like a simple change).

**Comment by lloydowen8** [comment]
Yeah this is an old issue and no longer relevant. The images are now saved into corresponding folders. 

Saving as PNG images is probably something that can be done to save space. The image arrays are currently in floating point representation (between 0-1). These values need to be changed to fall in the range 0-255 and converted to `uint8` before saving.
